Assets/ValueCalc.cs

```diff
@@ -0,0 +1,80 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ValueCalc : MonoBehaviour
+{
+    // Start is called before the first frame update
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        float angleX = gameObject.transform.rotation.eulerAngles.x;
+        float angleY = gameObject.transform.rotation.eulerAngles.y;
+        float angleZ = gameObject.transform.rotation.eulerAngles.z;
+
+        float angleXZ = (angleX + angleZ) % 360;
+
+        int orientation = 0;
+        
+        if(FuzzyBounds(angleX, 90))
+        {
+            orientation = 2;
+        }
+        else if(FuzzyBounds(angleX, 270))
+        {
+            orientation = 5;
+        }
+        else if(FuzzyBounds(angleXZ, 0))
+        {
+            orientation = 3;
+        }
+        else if(FuzzyBounds(angleXZ, 90))
+        {
+            orientation = 6;
+        }
+        else if(FuzzyBounds(angleXZ, 180))
+        {
+            orientation = 4;
+        }
+        else if(FuzzyBounds(angleXZ, 270))
+        {
+            orientation = 1;
+        }
+
+        Debug.Log(orientation);
```

Comment:
Can orientation still be 0 at this stage?

Reply:
Yeah. 0 is like a default value. If the die is rolling or on its edge (unlikely to happen but it can) we don't have a definitive value so it just says 0